### ISSUE TITLE
Exception in new socket code after some time

### ISSUE THREAD

**Issue by pavoni** [pavoni/pyloopenergy#33]
```
python demo.py                                                                                                                  ⏎
Electricity = 0.456
Gas = 0.0
Electricity = 0.453
Electricity = 0.46
Electricity = 0.456
Exception in thread LoopEnergy Event Thread:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/greg/ha/pyloopenergy/pyloopenergy/loop_energy.py", line 168, in _run_event_thread
    socket_io.wait(seconds=WAIT_BEFORE_POLL)
  File "/Users/greg/ha/pyloopenergy/env/lib/python3.7/site-packages/socketIO_client_nexus/__init__.py", line 251, in wait
    self._process_packets()
  File "/Users/greg/ha/pyloopenergy/env/lib/python3.7/site-packages/socketIO_client_nexus/__init__.py", line 276, in _process_packets
    for engineIO_packet in self._transport.recv_packet():
  File "/Users/greg/ha/pyloopenergy/env/lib/python3.7/site-packages/socketIO_client_nexus/transports.py", line 158, in recv_packet
    packet_text)
  File "/Users/greg/ha/pyloopenergy/env/lib/python3.7/site-packages/socketIO_client_nexus/parsers.py", line 96, in parse_packet_text
    packet_type = int(get_character(packet_text, 0))
  File "/Users/greg/ha/pyloopenergy/env/lib/python3.7/site-packages/socketIO_client_nexus/symmetries.py", line 33, in get_character
    return chr(get_byte(x, index))
  File "/Users/greg/ha/pyloopenergy/env/lib/python3.7/site-packages/socketIO_client_nexus/symmetries.py", line 29, in get_byte
    return indexbytes(x, index)
IndexError: index out of range
```

**Comment by warrenseymour** [comment]
Any update on this? I'm seeing Home Assistant shutdown whenever Loop's API service is unavailable - typically happens in the early hours of the morning, but I've just had it happen during prime evening hours. I'd really like to keep the Loop integration installed, but may have to find an alternative if this can't be fixed :(

Would love to pitch in, but Python isn't my bag.

**Comment by pavoni** [comment]
I have a patch that seems to work.
Will release it over the next few days.

**Comment by pavoni** [comment]
Now released as `0.1.3`
